### PR TITLE
Update the version number and release notes for v2.11.0rc2

### LINKED ISSUE #2461: Update the version number and release notes for v2.11.0rc2
See also:

+ #2410
+ #2414 (ensure https://github.com/ESMValGroup/ESMValCore/pull/2414#discussion_r1598833593 is addressed)

### PR DESCRIPTION
**Pull request by ehogan**
## Description

Closes #2461 

Link to documentation: https://esmvaltool--2462.org.readthedocs.build/projects/ESMValCore/en/2462/changelog.html#v2-11-0

***

## [Before you get started](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#getting-started)

- [x] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [x] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [x] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)
- [x] [🛠][1] [Documentation](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#documentation) is available
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

**Comment by ehogan** [comment]
Running the `draft_release_notes.py` script gave me the following changes:

```diff
diff --git a/doc/changelog.rst b/doc/changelog.rst
index e3e4b518d..4840c5b28 100644
--- a/doc/changelog.rst
+++ b/doc/changelog.rst
@@ -19,6 +19,7 @@ Backwards incompatible changes
 TODO: add examples of how to deal with these changes
 
 -  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+-  Use `iris.FUTURE.save_split_attrs = True` to remove iris warning (:pull:`2398`) by :user:`schlunma`

 Deprecations
 ~~~~~~~~~~~~
@@ -33,6 +34,9 @@ Bug fixes
 -  Do not overwrite facets from recipe with CMOR table facets for derived variables (:pull:`2255`) by :user:`bouweandela`
 -  Fix error message in variable definition check (:pull:`2313`) by :user:`enekomartinmartinez`
 -  Unify dtype handling of preprocessors (:pull:`2393`) by :user:`schlunma`
+-  Fix bug in `_rechunk_aux_factory_dependencies` (:pull:`2428`) by :user:`ehogan`
+-  Avoid loading entire files into memory when downloading from ESGF (:pull:`2434`) by :user:`bouweandela`
+-  Preserve cube attribute global vs local when concatenating (:pull:`2449`) by :user:`bouweandela`
 
 CMOR standard
 ~~~~~~~~~~~~~
@@ -55,6 +59,9 @@ Computational performance improvements
 -  Cache regridding weights if possible (:pull:`2344`) by :user:`schlunma`
 -  Implement lazy area weights (:pull:`2354`) by :user:`schlunma`
 -  Avoid large chunks in :func:`esmvalcore.preprocessor.climate_statistics` preprocessor function with `period='full'` (:pull:`2404`) by :user:`bouweandela`
+-  Load data only once for ESMPy regridders (:pull:`2418`) by :user:`bouweandela`
+-  Make volume calculation in preprocessor function `volume_statistics` lazy (:pull:`2436`) by :user:`bouweandela`
+-  Lazy mask_fillvalues preprocessor function (:pull:`2340`) by :user:`bouweandela`
 
 Documentation
 ~~~~~~~~~~~~~
@@ -65,6 +72,7 @@ Documentation
 -  Fix Codacy badge (:pull:`2382`) by :user:`bouweandela`
 -  Change 'mean' to 'percentile' in doc strings of preprocessor statistics (:pull:`2327`) by :user:`lukruh`
 -  Fixed typo in doc about weighted statistics (:pull:`2387`) by :user:`schlunma`
+-  Add advice on configuring Dask (:pull:`2435`) by :user:`bouweandela`
 
 Fixes for datasets
 ~~~~~~~~~~~~~~~~~~
@@ -88,6 +96,11 @@ Installation
 -  Pin pandas yet again avoid new ``2.2.1`` as well (:pull:`2353`) by :user:`valeriupredoi`
 -  Update Iris pin to avoid using versions with memory issues (:pull:`2408`) by :user:`chrisbillowsMO`
 -  Pin esmpy <8.6.0 (:pull:`2402`) by :user:`valeriupredoi`
+-  Pin numpy<2.0.0 to avoid pulling 2.0.0rcX (:pull:`2415`) by :user:`valeriupredoi`
+-  Remove compilers dependency from conda environment and force install x64 git in OSX Github Action workflows (:pull:`2420`) by :user:`valeriupredoi`
+-  Add support for Python=3.12 (:pull:`2228`) by :user:`valeriupredoi`
+-  Updated esmf-related pins (:pull:`2445`) by :user:`schlunma`
+-  Added iris-grib depedency (:pull:`2453`) by :user:`schlunma`
 
 Preprocessor
 ~~~~~~~~~~~~
@@ -105,6 +118,7 @@ Preprocessor
 -  Modernize :func:`esmvalcore.preprocessor.regrid_time` and allow setting a common calendar for decadal, yearly, and monthly data (:pull:`2311`) by :user:`schlunma`
 -  Added unstructured linear regridding (:pull:`2350`) by :user:`schlunma`
 -  Add preprocessors :func:`esmvalcore.preprocessor.distance_metrics` and :func:`esmvalcore.preprocessor.histogram` (:pull:`2299`) by :user:`schlunma`
+-  Add `interpolate` to `resample_hours` (:pull:`2440`) by :user:`schlunma`
 
 Automatic testing
 ~~~~~~~~~~~~~~~~~
@@ -121,6 +135,7 @@ Automatic testing
 -  Add calls to ``conda list`` in Github Action test workflows to inspect environment (:pull:`2391`) by :user:`valeriupredoi`
 -  Pin pandas yet again :panda_face: ``test_icon`` fails again with pandas=2.2.2 (:pull:`2394`) by :user:`valeriupredoi`
 -  Fixed units of cl test data (necessary since iris>=3.8.0) (:pull:`2403`) by :user:`schlunma`
+-  [CI Github Actions] Update (outdated) actions versions that produce Node.js warnings (:pull:`2411`) by :user:`valeriupredoi`
 
 Improvements
 ~~~~~~~~~~~~
@@ -132,12 +147,14 @@ Improvements
 -  Handle warnings about invalid units for iris>=3.8 (:pull:`2378`) by :user:`schlunma`
 -  Added note on how to access ``index.html`` on remote server (:pull:`2276`) by :user:`schlunma`
 -  Remove custom fix for concatenation of aux factories now that bug in iris is solved (:pull:`2392`) by :user:`schlunma`
--  Use ``iris.FUTURE.save_split_attrs = True`` to remove iris warning (:pull:`2398`) by :user:`schlunma`
 -  Ignored iris warnings about global attributes (:pull:`2400`) by :user:`schlunma`
+-  Increase the version number for 2.11.0 release (:pull:`2410`) by :user:`chrisbillowsMO`
+-  Add draft release notes for v2.11.0rc1 (:pull:`2414`) by :user:`chrisbillowsMO`
+-  Add native6, OBS6 and RAWOBS rootpaths to metoffice config-user.yml template, and remove temporary dir (:pull:`2432`) by :user:`alistairsellar`
+-  Make DRS path templates configurable per rootpath (:pull:`1894`) by :user:`bouweandela`
 
 .. _changelog-v2-10-0:
 
-
 v2.10.0
 -------
 Highlights
```

**Comment by ehogan** [comment]
Removed:

> Updated iris pin to ``iris>=3.6.1`` (:pull:`2286`) by :user:`schlunma`

as discussed at https://github.com/ESMValGroup/ESMValCore/pull/2414#pullrequestreview-2052987637 in 1f7d368a7

**Comment by ehogan** [comment]
Added highlights from #2412 via b3d715c4a.

File: doc/changelog.rst [hunk 1]
```diff
@@ -19,6 +30,7 @@ Backwards incompatible changes
 TODO: add examples of how to deal with these changes
 
 -  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+-  Use ``iris.FUTURE.save_split_attrs = True`` to remove iris warning (:pull:`2398`) by :user:`schlunma`
```
Comment (ehogan) [inline doc/changelog.rst:33]: @rebeccaherman1 and @schlunma, would it be possible for you to provide some examples of how to deal with these backwards incompatible changes, please? 😊

Comment (bouweandela) [reply doc/changelog.rst:33]: And also upgrade advice for the deprecations please

**Comment by chrisbillowsMO** [comment]
Per @valeriupredoi's suggestion [here](https://github.com/ESMValGroup/ESMValCore/discussions/2412#discussioncomment-9445893) I've added a couple of lines highlighting the Met Office's first time as release manager here: a4eb6eb

We've tried to walk the line of using it to celebrate the thriving ESMValTool community - and not just pat ourselves on the back.   Hopefully that comes across!

**Comment by valeriupredoi** [comment]
> Per @valeriupredoi's suggestion [here](https://github.com/ESMValGroup/ESMValCore/discussions/2412#discussioncomment-9445893) I've added a couple of lines highlighting the Met Office's first time as release manager here: [a4eb6eb](https://github.com/ESMValGroup/ESMValCore/commit/a4eb6eb99fa7dfd6b1b60e6a91dc1c885cd0c86e)
> 
> We've tried to walk the line of using it to celebrate the thriving ESMValTool community - and not just pat ourselves on the back. Hopefully that comes across!

looks great and well-deserved :beer: x3

File: doc/changelog.rst (hunk 1 above)
Comment (ehogan) [reply doc/changelog.rst:39]: Thanks @schlunma! 🎉 I have added this 4f61b8e35, see also https://esmvaltool--2462.org.readthedocs.build/projects/ESMValCore/en/2462/changelog.html#backwards-incompatible-changes (when it has finished building) 👍

Comment (ehogan) [reply doc/changelog.rst:39]: I haven't heard from @rebeccaherman1, so I was wondering whether the reviewer (@valeriupredoi) would have the knowledge to provide some details for this? 🤞

Comment (bouweandela) [reply doc/changelog.rst:39]: Thanks @schlunma! Could you please also add upgrading advice for #2231, which is listed under deprecations?

Comment (ehogan) [reply doc/changelog.rst:39]: Just did this via 8c492056d 😁

Comment (bouweandela) [reply doc/changelog.rst:39]: Thanks!

File: doc/changelog.rst [hunk 2]
```diff
@@ -19,11 +36,33 @@ Backwards incompatible changes
 TODO: add examples of how to deal with these changes
```
Comment (bouweandela) [inline doc/changelog.rst:36]: Remove this line?

File: doc/changelog.rst [hunk 3]
```diff
@@ -19,11 +36,33 @@ Backwards incompatible changes
...
 
 -  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
```
Comment (bouweandela) [inline doc/changelog.rst:38]:
```suggestion
-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
   The preprocessor function :func:`esmvalcore.preprocessor.extract_region` no longer
   automatically maps the extracted :class:`iris.cube.Cube` to the 0-360 degrees 
   longitude domain. If you need this behaviour, use
   `cube.intersection(longitude=(0., 360.))` in your Python code after extracting the
   region. There is no possibility to restore the previous behaviour from a recipe.
```
Maybe add something along those lines?

**Review by bouweandela — CHANGES_REQUESTED** [review]

File: doc/changelog.rst (hunk 2 above)
Comment (ehogan) [reply doc/changelog.rst]: Done in 5eaa87c1e 😊

File: doc/changelog.rst (hunk 3 above)
Comment (ehogan) [reply doc/changelog.rst:36]: Thanks @bouweandela! 🎉 Done in 86950411c 😊

**Comment by ehogan** [comment]
Ok, I think that's everything! 🤞

**Comment by valeriupredoi** [comment]
absolute legends yous! Lemme do a final check, and I can also merge if peeps are ready-ready :beer:

**Review by valeriupredoi — APPROVED** [review]
:beers: :partying_face:

**Review by bouweandela — APPROVED** [review]

**Comment by ehogan** [comment]
Thanks everyone! 🥳 🍻

**Pull request merged**